drivers/crypto/crypto_rts5912_priv.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2025 Realtek Semiconductor Corporation, SIBG-SD7
+ *
+ */
+
+#ifndef ZEPHYR_DRIVERS_CRYPTO_CRYPTO_RTS5912_PRIV_H_
+#define ZEPHYR_DRIVERS_CRYPTO_CRYPTO_RTS5912_PRIV_H_
+
+struct rts5912_sha256_context {
+	uint32_t total[2];
+	uint32_t state[8];
+	uint8_t buffer[64];
+	bool is224;
+	uint8_t sha2_data_in_sram[1024];
+} __packed;
+
+struct rts5912_sha_config {
+	volatile struct sha2_type *cfg_sha2_regs;
+	volatile struct sha2dma_type *cfg_sha2dma_regs;
+} __packed;
```

Comment:
The `__packed` attribute needs to be on the structure that defines the hardware registers.  When you add `__packed` here, you only force packing on `rts5912_sha_config`, and not on `struct sha2_type` or `struct sha2dma_type`

```suggestion
};
```